sdk/inc/azure/core/az_version.h

```diff
@@ -17,19 +17,19 @@
 
 /// The version in string format used for telemetry following the `semver.org` standard
 /// (https://semver.org).
-#define AZ_SDK_VERSION_STRING "1.3.2"
+#define AZ_SDK_VERSION_STRING "1.4.0-beta.1"
 
 /// Major numeric identifier.
 #define AZ_SDK_VERSION_MAJOR 1
 
 /// Minor numeric identifier.
-#define AZ_SDK_VERSION_MINOR 3
+#define AZ_SDK_VERSION_MINOR 4
 
 /// Patch numeric identifier.
-#define AZ_SDK_VERSION_PATCH 2
+#define AZ_SDK_VERSION_PATCH 0
 
 /// Optional pre-release identifier. SDK is in a pre-release state when present.
-#define AZ_SDK_VERSION_PRERELEASE
-#undef AZ_SDK_VERSION_PRERELEASE
+#define AZ_SDK_VERSION_PRERELEASE "beta.1"
+#define AZ_SDK_VERSION_PRERELEASE "beta.1"
```

Comment:
We have to fix this with a manual update, just like https://github.com/Azure/azure-sdk-for-c/pull/2171

The engsys script doesn't address the version update correctly after a GA release. Here's the issue tracking that work, @RickWinter:
https://github.com/Azure/azure-sdk-for-c/issues/2108

Reply:
Fixed in https://github.com/Azure/azure-sdk-for-c/pull/2276